django/mgmt/views.py

```diff
@@ -211,6 +213,8 @@ def get(self, request):
 
     def post(self, request):
         try:
+            if request.user.username == PUBLIC_ACCESS_USERNAME:
```

Comment:
is there any case where we'd have multiple usernames in the future (`PUBLIC_ACCESS_USERNAME: List[str]`?)

Reply:
I am seeing it as public is public.  Otherwise we'd probably make them login themselves and give them rights to other items.  We can discuss at our standup meeting..